_health-care/_js/utils/validations.js

```diff
@@ -190,7 +192,20 @@ function isValidSection(completePath, sectionData) {
   }
 }
 
+function initializeNullValues(value) {
+  if (value === null) {
+    return '';
+  } else if (_.isPlainObject(value)) {
```

Comment:
I'm not sure about this, but when I was testing things locally, I first noticed that arrays were returning true for `typeof (value) === 'object'`, so when I was testing checking if something was an array, I checked that first before then checking if it was (or wasn't) an object.